entry_types/scrolled/package/src/entryState/legalInfo.js

```diff
@@ -0,0 +1,81 @@
+import {useEntryMetadata} from "./metadata";
+import {useEntryState} from "./EntryStateProvider";
+import {getItems} from "../collections";
+import {useMemo} from "react";
+
+/**
+ * Returns a string (comma-separated list) of copyrights of
+ * all images used in the entry.
+ * If none of the images has a rights attribute configured,
+ * it falls back to the default file rights of the entry's account,
+ * otherwise returns an empty string
+ *
+ * @example
+ *
+ * const fileRights = useFileRights();
+ * fileRights // => "author of image 1, author of image 2"
+ */
+export function useFileRights() {
+  const entryState = useEntryState();
+
+  const defaultFileRights = entryState.config.defaultFileRights;
+  const imageFiles = getItems(entryState.collections, 'imageFiles');
+  const imageFileRights = imageFiles.map(function (imageConfig) {
+    return imageConfig.rights ? imageConfig.rights.trim() : undefined;
+  }).filter(Boolean).join(', ');
+  const fileRights = !!imageFileRights ? imageFileRights : defaultFileRights.trim();
+  const fileRightsString = !!fileRights ? ('Bildrechte: ' + fileRights) : '';
+
+  return useMemo(() => {
+    return fileRightsString;
+  }, [imageFileRights]);
```

Comment:
Just as an aside, this `useMemo` does nothing since the dependency, `imageFileRights`, is a new object every time (constructed by the `map` call a few lines up). It also doesn't hurt, though.